_episodes/07-testing.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Testing"
+teaching: 0
+exercises: 0
+questions:
+- "How can I know that my package works as expected?"
+objectives:
+- "Understand why tests are useful"
+- "Learn how to create a battery of basic tests"
+- "Learn how to run the tests"
+keypoints:
+- "FIXME"
+---
+
+What is the first thing you do after writing a function or another piece of code?
+Most likely, you try it, and figure out if it works as expected.
+Right?
+If the answer is yes, congratulations!
+You already know the very basics of testing!
+
+The main lesson we'll get out of this episode is that those kind of tests are extremely valuable.
+So much, that they deserve more than just be typed on the console and be forgotten forever.
+Tests deserve to be saved, and even more, rerun often.
+
+## Why shall I store my tests?
+
+The vast majority of people needs to try their scripts and functions in order to make sure they work fine.
+And we do it.
+What is the added value of storing those tests?
+> ## What is the added value of storing my tests?
+> Storing tests in the form of a script takes a bit more time and effort than just running them interactively in the console.
+> 
```

Comment:
Should the participants not know what a test is before they can answer this question?

Reply:
That's a good one. I am trying to let the reader naturally drift between the intuitive, somehow loose concept of _"test"_ into the technical one of _"unit test with testthat"_. I'll think about it.